src/libponyrt/CMakeLists.txt

```diff
@@ -150,7 +151,7 @@ if(PONY_RUNTIME_BITCODE)
     list(APPEND _ll_bc_all ${_ll_except_src})
 
     add_custom_target(libponyrt_bc ALL
-        llvm-link -o "${libponyrt_BINARY_DIR}/libponyrt.bc" ${_ll_bc_all}
+        ${libponyrt_SOURCE_DIR}/../../build/libs/bin/llvm-link -o "${libponyrt_BINARY_DIR}/libponyrt.bc" ${_ll_bc_all}
```

Comment:
why do we need llvm-link now and we didn't before?

Reply:
llvm-link was needed before this PR for building the runtime bitcode, and it is still needed after this PR for building the runtime bitcode.

The change made here makes it so that we are using the `llvm-link` binary that we built in the earlier step, rather than just whatever random version of `llvm-link` may (or may not) be present on the system. The correct version must be used - or rather, the version used must include WebAssembly, and it must be at or greater than the LLVM version that we built in `lib`.

The easiest and cleanest thing to do is just to use the LLVM that we built in lib - which is what this change does.

Reply:
Makes sense. I wonder if we are relying on other tools we should be using from the llvm we built.

Reply:
It's likely that the main reason we didn't catch this one is because the runtime bitcode build is not very widely exercised - I don't think it's done in CI and not very many people have done it personally either.